plugins/woocommerce/includes/admin/class-wc-admin-menus.php

```diff
@@ -319,8 +319,8 @@ public function addons_page() {
 	 */
 	public function orders_menu(): void {
 		if ( wc_get_container()->get( CustomOrdersTableController::class )->custom_orders_table_usage_is_enabled() ) {
-			$this->orders_page_controller = wc_get_container()->get( Custom_Orders_PageController::class );
-			$this->orders_page_controller->setup();
+			$orders_page_controller = wc_get_container()->get( Custom_Orders_PageController::class );
+			$orders_page_controller->setup();
```

Comment:
This could be reduced to the single line to be consistent with the `else` code.

```suggestion
			wc_get_container()->get( Custom_Orders_PageController::class )->setup();
```

Reply:
I've applied your suggestion in f11152c0bc37d7320e44822b17add3220f2d33bd. Thanks @rrennick!